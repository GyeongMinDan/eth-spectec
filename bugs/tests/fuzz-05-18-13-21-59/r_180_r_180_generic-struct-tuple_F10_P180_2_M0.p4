struct S<T> {
  tuple<T, tuple<match_kind, int>> t;
}
