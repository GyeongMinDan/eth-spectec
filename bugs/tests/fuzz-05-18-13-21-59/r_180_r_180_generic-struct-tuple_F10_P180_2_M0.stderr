likely_hits/r_180_r_180_generic-struct-tuple_F10_P180_2_M0.p4(2): [--Werror=type-error] error: Field 'f1' of 'struct tuple_0' cannot have type 'int'
  tuple<T, tuple<match_kind, int>> t;
                             ^^^
[--Werror=type-error] error: Error while analyzing struct tuple_0
In file: /p4c/lib/crash.cpp:299
Compiler Bug: Exiting with SIGSEGV

